src/core/api/index.ts

```diff
@@ -2010,7 +2010,17 @@ class Player extends EventEmitter<PLAYER_EVENT_STRINGS, any> {
       default:
         const adaptations = period.adaptations[type];
         if (adaptations && adaptations.length) {
-          adaptation$.next(adaptations[0]);
+          const mainAdaptation = adaptations.find((adaptation) => {
```

Comment:
Not sure we should use directly the schemeIdURI in the ``core``.

The manifest structure here should be generic enough.

What about still using the first, but manage to put the main adaptation first in ``src/parsers/manifest/dash``?

That or defining a new single property.